src/MSA.BuildingBlocks.ServiceClient/README.md

```diff
@@ -11,29 +11,30 @@ To install the package, use the following command in the Package Manager Console
     PM> Install-Package MSA.BuildingBlocks.ServiceClient
 
 ## Usage
-
-First, add the `IServiceClient` to the dependency injection container in `Startup.cs`:
+<details open> <summary>For package version 0.1.0</summary>
+    
+First, add the `ServiceClient` to the dependency injection container in `Startup.cs`:
```

Comment:
add the `ServiceClient` to the dependency but 
![image](https://github.com/Marusyk/MSA.BuildingBlocks/assets/9221694/d793edff-59a7-4ba2-9b71-9e21e2db2d34)
you replaced AddServiceClient to AddHttpClient. is that correct?

Reply:
For 0.1.0 version is correct. I am going to change it.